packages/xrpl/test/client/getOrderbook.test.ts

```diff
@@ -92,47 +102,76 @@ describe('client.getOrderbook', function () {
     assertResultMatch(response, expectedResponse, 'getOrderbook')
   })
 
-  it('invalid options', async function () {
-    this.mockRippled.addResponse('book_offers', normalRippledResponse)
-    assertRejects(
-      this.client.getOrderbook(
-        requests.getOrderbook.normal.takerPays,
-        requests.getOrderbook.normal.takerGets,
-        {
-          invalid: 'options',
-        },
+  it('invalid options', async () => {
+    const invalidOptions = [
+      {
+        option: 'invalid',
+      },
+      {
+        limit: 'invalid',
+      },
+      {
+        ledger_index: 'invalid',
+      },
+      {
+        ledger_hash: 0,
+      },
+      {
+        taker: 0,
+      },
+    ]
+
+    testContext.mockRippled!.addResponse('book_offers', normalRippledResponse)
+    await Promise.all(
+      invalidOptions.map(
+        async (options) =>
```

Comment:
nit: `option` might be better named `invalidOption` (Took me a second to understand where `options` came from - I thought it was coming from a similar spot to the `requests.getOrderbook.normal.takerGets`)

Reply:
Fixed